src/main/java/com/paneedah/weaponlib/inventory/CustomPlayerInventoryGuiContainer.java

```diff
@@ -22,6 +22,7 @@ public CustomPlayerInventoryGuiContainer(EntityPlayer player, InventoryPlayer in
     @Override
     public void initGui() {
         super.initGui();
+        this.buttonList.clear();
```

Comment:
1. unnecessary
2. Please do not touch anything related to the equipment system as it's being reworked in #138

Reply:
This is a statement that was once included, you may have forgotten to add it